mail/mailer.go

```diff
@@ -32,12 +38,31 @@ func New(server, port, username, password string) MailerImpl {
 		Port:   port,
 		Auth:   auth,
 		From:   username,
+		clk:    clock.Default(),
+	}
+}
+
+func (m *MailerImpl) generateMessage(to []string, subject, body string) []byte {
+	now := m.clk.Now().UTC()
+	rand.Seed(int64(now.Nanosecond()))
+	headers := []string{
+		fmt.Sprintf("To: %s", strings.Join(to, ", ")),
```

Comment:
Won't we need to do some kind of escaping if we've accidentally let in special characters we shouldn't have? Commas and the ilk.

This feels very, very low-level. Is there a higher level library that could help us out?